doc/man1/openssl-s_server.pod

```diff
@@ -327,7 +329,9 @@ is in use.
 
 =item B<-dcertform> B<DER>|B<PEM>, B<-dkeyform> B<DER>|B<PEM>, B<-dpass> I<val>
 
-Additional certificate and private key format and passphrase respectively.
+The format of the certificate, private key, and passphrase for the
+additional cert; the default format is B<PEM>.
+see L<openssl(1)/Pass Phrase Options>.
```

Comment:
Perhaps separate the dpass or add both links to format options and pass phrase options.